deploy/004_oracle_aggregator.ts

```diff
@@ -7,9 +7,9 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
   const { deployer, governor: superAdmin } = await hre.getNamedAccounts();
 
   const identityOracle = await hre.deployments.get('IdentityOracle');
-  const chainlinkAdapter = await hre.deployments.get('StatefulChainlinkOracleAdapter');
+  const chainlinOracle = await hre.deployments.get('StatefulChainlinkOracle');
```

Comment:
Should we keep the `StatefulChainlinkOracle` name?

Reply:
We are keeping it... What do you mean?

Reply:
I mean, should we change it? Hahaha. That was what I was insinuating.

Reply:
Change it to what? It's a Chainlink Oracle they is stateful 

I don't understand what you want me to do

Reply:
Nevermind Sam 😂  it's not that important. it was a misc question.